src/features/PopularMovies/MovieDetails/Header/index.js

```diff
@@ -13,9 +15,27 @@ import {
 
 export const Header = () => {
     const movieData = useSelector(selectMovieData);
+    const configuration = useSelector(selectConfiguration);
 
-    const imageURL = "http://image.tmdb.org/t/p/";
-    const size = "original";
+    const [mQuery, setMQuery] = useState({
+        matches: window.innerWidth > 1280 ? true : false
+    });
+
+    useEffect(() => {
+        const mediaQuery = window.matchMedia("(min-width: 1280px)");
+        mediaQuery.addListener(setMQuery);
+        return () => mediaQuery.removeListener(setMQuery);
+      }, [mQuery]);
+
+      let size;
+
+      if (mQuery.matches) {
+        size = configuration.images.backdrop_sizes[3];
+      } else {
+        size = configuration.images.backdrop_sizes[2];
+      }
```

Comment:
It seems to work great! 👍   

But:
- Formatting is needed.
- Let's use theme.breakpoints for screen width.
- Having `mQuery` and `mediaQuery` names is a bit unclear to me. Is there any other way we can call them? For example `mediaQuery` we can change to `MediaQueryList`. That's what window.machMedia() returns acording to MDN [https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia](url)
- I haven't managed to wrap my mind around the logic you wrote here yet but do you actually need `[mQuery]` in `useEffect()`?

Reply:
i changed it - now it works in all sizes of posters